### PR TITLE
Riya/slash modal/test

### PR DESCRIPTION
**Pull request by ViaSocket-Git**

### CODE REVIEW

**Review by Saurabh186 — CHANGES_REQUESTED** [review]

**Review by Saurabh186 — CHANGES_REQUESTED** [review]

File: src/components/tiptapEditor/tiptap.scss [hunk 1]
```diff
@@ -23,6 +23,57 @@
...
+
+    ul {
+        list-style: none;
+        width: 100%;
+        max-height: 300px;
+
+        li {
+            padding: 6px 15px;
+            transition: background-color 0.2s;
+            box-sizing: border-box;
```
Comment (Saurabh186) [inline src/components/tiptapEditor/tiptap.scss:43]: use kya hai iska

File: src/components/tiptapEditor/tiptap.scss [hunk 2]
```diff
@@ -23,6 +23,57 @@
...
+
+            &:focus {
+                outline: none;
+                background-color: #f4f4f4;
```
Comment (Saurabh186) [inline src/components/tiptapEditor/tiptap.scss:47]: use variables

File: src/components/tiptapEditor/tiptap.scss [hunk 3]
```diff
@@ -23,6 +23,57 @@
...
+            }
+
+            &:hover {
+                background-color: #f4f4f4;
```
Comment (Saurabh186) [inline src/components/tiptapEditor/tiptap.scss:51]: use variables

File: src/components/tiptapEditor/tiptap.scss [hunk 4]
```diff
@@ -23,6 +23,57 @@
...
+            }
+
+            .slash-menu-icon {
+                margin-right: 22px;
```
Comment (Saurabh186) [inline src/components/tiptapEditor/tiptap.scss:55]: mr-4 =24px

File: src/components/tiptapEditor/tiptap.scss [hunk 5]
```diff
@@ -23,6 +23,57 @@
...
+                margin-left: 10px;
```
Comment (Saurabh186) [inline src/components/tiptapEditor/tiptap.scss:56]: ml-2 = 8px

File: src/components/tiptapEditor/tiptap.scss [hunk 6]
```diff
@@ -23,6 +23,57 @@
...
+    padding: 10px 0;
+    z-index: 1;
+
+    ul {
+        list-style: none;
+        width: 100%;
+        max-height: 300px;
+
+        li {
+            padding: 6px 15px;
```
Comment (Saurabh186) [inline src/components/tiptapEditor/tiptap.scss:41]: py-1 px-3

File: src/components/tiptapEditor/tiptap.scss [hunk 7]
```diff
@@ -23,6 +23,57 @@
...
+                background-color: #f4f4f4;
+            }
+
+            .slash-menu-icon {
+                margin-right: 22px;
+                margin-left: 10px;
+            }
+
+            .menu-label {
+                font-weight: 500;
```
Comment (Saurabh186) [inline src/components/tiptapEditor/tiptap.scss:60]: fw-500

File: src/components/tiptapEditor/tiptap.scss [hunk 8]
```diff
@@ -23,6 +23,57 @@
...
+                font-size: 14px;
```
Comment (Saurabh186) [inline src/components/tiptapEditor/tiptap.scss:61]: font-14

File: src/components/tiptapEditor/tiptap.scss [hunk 9]
```diff
@@ -23,6 +23,57 @@
...
+            }
+
+            .menu-description {
+                font-size: 12px;
```
Comment (Saurabh186) [inline src/components/tiptapEditor/tiptap.scss:65]: font-12

File: src/components/tiptapEditor/tiptap.scss [hunk 10]
```diff
@@ -23,6 +23,57 @@
...
+                color: #888;
```
Comment (Saurabh186) [inline src/components/tiptapEditor/tiptap.scss:66]: use variables

**Review by Saurabh186 — DISMISSED** [review]
jo class nahi hai usko scss me likhna hai bs

**Review by goutam01mehta — APPROVED** [review]

**Pull request merged**